frontend/src/metabase/visualizations/lib/apply_tooltips.js

```diff
@@ -32,11 +32,13 @@ export function getClickHoverObject(
   const isBar = classList.includes("bar");
   const isSingleSeriesBar = isBar && !isMultiseries;
 
-  function getColumnDisplayName(col) {
-    const title = getIn(settings, ["series_settings", col.name, "title"]);
+  function getColumnDisplayName(col, colIndex, card) {
+    const colKey = seriesIndex > 0 && colIndex === 1 ? card.name : col.name;
```

Comment:
I think this line warrants a comment above it to explain why we need to do this

Reply:
Agree, added a comment